futurenhs.api/FutureNHS.Api/Controllers/FolderController.cs

```diff
@@ -61,22 +62,35 @@ public async Task<IActionResult> CreateFolderAsync(Guid userId, string slug, Fol
         }
 
         [HttpPost]
-        [Route("users/{userId:guid}/groups/{slug}/folders/{id:guid}")]
-        public async Task<IActionResult> CreateFolderAsync(Guid userId, string slug, Guid id, Folder folder, CancellationToken cancellationToken)
-        {
+        [Route("users/{userId:guid}/groups/{slug}/folders/{folderId:guid}")]
+        public async Task<IActionResult> CreateFolderAsync(Guid userId, string slug, Guid folderId, Folder folder, CancellationToken cancellationToken)
+        {            
             var sanitisedFolder = new Folder
```

Comment:
Unused variable unless we still need to sanitize? If so replace 'folder' on line 74 with 'sanitisedFolder'

Reply:
Ah good spot! we still need to sanitise it